applications/luci-app-softether/root/usr/share/rpcd/acl.d/luci-app-softether.json

```diff
@@ -0,0 +1,16 @@
+{
+	"luci-app-softether": {
+		"description": "Grant access to softether management",
+		"read": {
+			"cgi-io": [ "exec" ],
+			"file": {
+				"/usr/libexec/vpncmd-call account-list": [ "exec" ]
+			}
+		},
+		"write": {
```

Comment:
I don't know it exactly, but is that right with the `write`? Is not a `read` sufficient?

Reply:
Moved to "read".